astro/src/content/docs/operate/secure-and-monitor/advanced-threat-detection.mdx

```diff
@@ -23,9 +23,7 @@ Flagging and responding to suspicious behavior is a part of any cybersecurity pr
 Advanced Threat Detection, a feature that brings best-practice functionality to help you deal with bizarre, possibly malicious behavior around logins, registrations, user creation, and user updates.
 
 <Aside type="note">
-Turning this feature on requires a support ticket, as it is being rolled out on an account by account basis.
-
-To enable Advanced Threat Detection, submit a [support ticket through your account portal](https://account.fusionauth.io/account/support) and request it be turned on.
+Advanced Threat Detection can be enabled when the user creates a license.
```

Comment:
Can it be enabled at any other time? Like can I update my existing license to enable this?

If not, what should I do if I need one?

Reply:
It's your PR :) . Please make whatever changes you deem appropriate.

Reply:
I have updated when ATD can be enabled.